code/datums/gamemode/role/syndicate.dm

```diff
@@ -136,6 +136,7 @@
 /datum/role/traitor/challenger
 	name = CHALLENGER
 	id = CHALLENGER
+	name_for_uplink = CHALLENGER
```

Comment:
and that broke job exclusive items for challengers

Reply:
oh I get it, you intended this for syndie deathsquads instead of challengers

Reply:
ohhh wait so that was specifically to exclude challengers from this bundle

Reply:
so yeah, better to add a separate check for roles than piggy-backing onto the job checks.